### ISSUE TITLE
test_must_gather[OTHERS] is failing

### ISSUE THREAD

**Issue by jilju** [red-hat-storage/ocs-ci#6181]
The test case test_must_gather[OTHERS] is failing in Manged services consumer cluster.

Error:
```
Error Details
Exception: Files don't exist:
['odf-csi-addons-operator.yaml']
Empty files:
[]
Content issues:
[]

```
RUN ID: 1657707533

**Comment by am-agrawa** [comment]
The failure is seen on multiple runs, pasting a few of them-
https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/4989/#showFailuresLink
ocs-ci results for OCS4-11-Downstream-OCP4-11-AWS-IPI-3AZ-RHCOS-3M-3W-tier1 (BUILD ID: 4.11.0-131 RUN ID: 1659951134)

and

https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/4987/
ocs-ci results for OCS4-11-Downstream-OCP4-12-VSPHERE6-UPI-1AZ-RHCOS-VSAN-3M-3W-tier1 (BUILD ID: 4.11.0-131 RUN ID: 1659954431)

**Comment by am-agrawa** [comment]
Also here-
https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/4960/
ocs-ci results for OCS4-11-Downstream-OCP4-11-AWS-UPI-Proxy-3AZ-RHCOS-3M-3W-tier1-or-tier_after_upgrade-post-upgrade (BUILD ID: 4.11.0-131 RUN ID: 1659892240)

**Comment by am-agrawa** [comment]
@OdedViner Pls look into it.

**Comment by OdedViner** [comment]
fixed here
https://github.com/red-hat-storage/ocs-ci/pull/6415/files#diff-9f05a4efcc76519e101869323e0579f0bee85c573ec08a82bf89ff97029440ecL728

**Comment by am-agrawa** [comment]
resolved #6415
